contentScript.js

```diff
@@ -509,12 +572,21 @@ const bindSubmitButtonToSaveInfo = () => {
     keeptryingcontinuously = document.getElementById("continuousretry").checked;
     enableAutoRefresh = document.getElementById("enableautorefresh").checked;
     enableautoconfirm = document.getElementById("enableautoconfirm").checked;
-    ageSelectorText = document.getElementById("ageselect").value;
     let searchPreftext = document.getElementById("searchpref").value;
     first_5_pin_digits = document.getElementById("pincodeinput").value;
     timeslotind = document.getElementById("timeslotinput").value;
     center_prefs_string = document.getElementById("centerprefinput").value;
     minavailability = document.getElementById("minavailabilityinput").value;
+    let temp_checked_buttons = [];
+    selected_button_checkbox = []
+    for (const key in buttonCheckboxMapping) {
+      let button_checkbox = document.getElementById(key);
+      buttonCheckboxMapping[key].checked = button_checkbox.checked;
+      if (button_checkbox.checked) {
+        selected_button_checkbox.push(key)
+        temp_checked_buttons.push(buttonCheckboxMapping[key].label)
```

Comment:
what are we doing with `temp_checked_buttons`

Reply:
Yikes! Will remove this!